app/components/rdf-input-fields/mandataris-fractie-selector.js

```diff
@@ -68,42 +69,37 @@ export default class MandatarisFractieSelector extends InputFieldComponent {
   }
 
   async loadProvidedValue() {
-    const membershipTriples = triplesForPath(this.storeOptions, false).values;
-    if (!membershipTriples.length) {
+    const fractieUris = triplesForPath(this.storeOptions, false).values;
+    if (!fractieUris.length) {
       return;
     }
-    const membershipUri = membershipTriples[0].value;
+    const fractieUri = fractieUris[0].value;
 
-    const matches = await this.store.query('lidmaatschap', {
-      'filter[:uri:]': membershipUri,
-      include: 'binnen-fractie',
+    const matches = await this.store.query('fractie', {
+      'filter[:uri:]': fractieUri,
     });
-    this.membership = matches.at(0);
+    this.selectedFractie = matches.at(0);
   }
 
+  // TODO Maybe turn into task?
   @action
   async onSelectFractie(fractie) {
     this.updating = true;
-    const newMembership = this.store.createRecord('lidmaatschap', {
-      binnenFractie: fractie,
-    });
-    await newMembership.save();
-    this.membership = newMembership;
+    const uri = fractie.uri;
```

Comment:
i think this will break when you clear the input. we should check if it's null
```suggestion
    const uri = fractie?.uri;
```